src/main/java/com/rabbitmq/perf/MulticastParams.java

```diff
@@ -294,7 +303,7 @@ public boolean shouldConfigureQueues() {
             generatedQueueNames.add(qName);
             // skipping binding to default exchange,
             // as it's not possible to explicitly bind to it.
-            if (!"".equals(exchangeName) && !"amq.default".equals(exchangeName)) {
+            if (!predeclared && !"".equals(exchangeName) && !"amq.default".equals(exchangeName)) {
```

Comment:
Is this change really related?

Reply:
Yes, because if you try to bind a sharded queue to an exchange you get an error.

Plus, if a queue is predeclared, allowing the perf-test tool to add another binding to it is probably not what you want.

Reply:
That makes sense but will break some scenarios, see c061038a62f6e0583d0ef4aaab6293bb8f4cdd97 and #43.

Reply:
@dmcintyre-pivotal do you think we should introduce a new boolean flag? I'm afraid this and #43 expect two completely different behaviours. `--skip-binding-queues`, perhaps?